NEFI_functions/worldclim2_grab.r

```diff
@@ -21,17 +22,18 @@
 #'                                                                                      2L))
 #' test.out <- worldclim2_grab(points[,2], points[,1])
 
-worldclim2_grab <- function(latitude,longitude,elev = 500, n.sim = 1000){
+worldclim2_grab <- function(latitude,longitude,elev = 500, n.sim = 1000,
+                            worldclim2_folder = '/fs/data3/caverill/WorldClim2/'){
   
   #make points an object
   points <- cbind(longitude, latitude)
  
   #load mean annual temperature and precipitation rasters from worldclim2
-  prec    <- raster::raster('/fs/data3/caverill/WorldClim2/wc2.0_bio_30s_12.tif')
-  temp    <- raster::raster('/fs/data3/caverill/WorldClim2/wc2.0_bio_30s_01.tif')
-  temp_CV <- raster::raster('/fs/data3/caverill/WorldClim2/wc2.0_bio_30s_04.tif')
-  prec_CV <- raster::raster('/fs/data3/caverill/WorldClim2/wc2.0_bio_30s_15.tif')
-  mdr     <- raster::raster('/fs/data3/caverill/WorldClim2/wc2.0_bio_30s_02.tif')
+  prec    <- raster::raster(paste0(worldclim2_folder,'wc2.0_bio_30s_12.tif'))
```

Comment:
For future reference, file.path is safer here than paste0, which can f* up depending on slashes and what not.